src/service/reviews/reviewService.ts

```diff
@@ -67,6 +68,7 @@ const updateReview = async (updatedReview: Review) => {
         throw e
     }
 }
+
 const flagReview = async (userId: number, businessId: number) => {
     try {
         const review: Review = reviewsRepo.findReview(userId, businessId)
```

Comment:
You're setting flagged = true. But Does this work in the database? Because the actual database column is inappropriate_flag. Also, you have to set a flag_reason (Why did you flag this review)